src/Repository/Projection/ReferentManagedUserRepository.php

```diff
@@ -199,6 +199,10 @@ private function createFilterQueryBuilder(Adherent $referent, ManagedUsersFilter
             $typeExpression->add($and);
         }
 
+        if ($filter->includeCP()) {
+            $typeExpression->add('u.citizenProjects IS NOT NULL or u.citizenProjectsOrganizer IS NOT NULL');
```

Comment:
what happens if the `citizenProjects` column has `[]` as value?

Reply:
1) 
```
$filter->getIncludeCitizenProject() && \count($filter->getIncludeCitizenProject()) > 0
```
the second condition is useless here, the first one checks that the array is not empty.


2) By my initial comment, I mean that the column `u.citizenProjects` can be `null` and can have `[]` as a valid JSON value and the rows with `[]` as a value will be selected with your request.